doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
+Bernhard Schommer, Matthieu Sozeau, Arnaud Spiwack, Paul Steckler,
+Enrico Tassi, Anton Trunov, Martin Vassor, Vadim Zaliva and Théo
+Zimmermann.
+
+The development process was coordinated by Matthieu Sozeau. Maxime Dénès
+was in charge of the release process. The maintainer of this release
+is ???.
+
+Many power users helped to improve the design of the new features via
+the bug tracker, the pull request system, the {\Coq} development mailing
+list or the coq-club mailing list. Special thanks to the users who
+contributed patches and intensive brain-storming and code reviews,
+starting with Jason Gross, Ralf Jung, Robbert Krebbers, Amin Timany,
+..???.  It would however be impossible to mention exhaustively the names
+of everybody who to some extent influenced the development.
+
+Version 8.8 is the third release of {\Coq} developed on a time-based
+development cycle. Its development spanned 6 months from the release of
+{\Coq} 8.7 and was based on a public road-map. It attracted many
+external contributions. Code reviews and continuous integration testing
+were systematically used before integration of new features, with an
+important focus given to compatibility and performance issues.
+
+Coq Enhancement Proposals (CEPs for short) and open pull-requests
+discussions were used to discuss publicly the new features. The
+developer's Coq Working Groups, which are systematically streamed on the
+Coq YouTube channel, are moving towards a faster frequency
+and experimenting videoconferencing technology, reflecting the
+increasingly multi-site nature of Coq development.
+
+The {\Coq} consortium, an organization directed towards users and
+supporters of the system, is now running and employs Maxime Dénès.
+The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
+
+The developers of {\Coq} are also authors of plugins that are highly
+related to the tool:
+\begin{itemize}
+\item Pierre-Marie Pédrot released the first version of Ltac 2.0, a
+  cleaned-up, typed redesign of {\Ltac} as a plugin, which is slated to
+  replace Ltac in the future.
+\item Enrico Tassi is developing the ELPI plugin, an embedding of λ-prolog
+  for logic programming of tactics or of the refinement engine.
+\item Abhishek Anand, Matthieu Sozeau, Simon Boulier and Nicolas
+  Tabareau are developing Template-Coq, a quoting library for {\Coq}
+  terms and kernel declarations which allows to script the
+  vernacular and write extensions to the system entirely in {\Coq}.
+\item Cyprien Mangin and Matthieu Sozeau released a new version of
+  \textsc{Equations}, a function definition package that aims to subsume
+  \textsc{Function} and \textsc{Program} in the future.
+\item Emilio Jesus Gallego Arias released a version of the jsCoq Online
+  Integrated Development Environmnent based on {\Coq} 8.8.  jsCoq is a
+  javascript port of {\Coq} that can be run directly in a web browser.
+\item ???
+\end{itemize}
+
+\begin{flushright}
+Santiago du Chili, March 2018,\\
```

Comment:
???

Reply:
The custom is to "sign" from where we are. And that's where I am actually.

Reply:
I quite don't see that this way:
- if you sing it alone (and I'm fine with it) write the place where you are
- if you sign it as the Coqdev team, given there is no single place where it is, remove the location

Reply:
(Also, I guess the tradition is to write the place where you work, not where you are the day you write the text)

Reply:
Ok